x-pack/plugins/ml/public/timeseriesexplorer/timeseries_chart.js

```diff
@@ -0,0 +1,1239 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+
+/*
+ * React component chart showing plot time series data, with or without model plot enabled,
```

Comment:
Should take the opportunity to reword this comment to say something like `React component chart plotting data from a single time series, with or without model plot enabled, annotated with anomalies.`